config_builder.go

```diff
@@ -869,6 +872,11 @@ type DatabaseInstances struct {
 	// WalletDB is the configuration for loading the wallet database using
 	// the btcwallet's loader.
 	WalletDB btcwallet.LoaderOption
+
+	// NativeSQLStore is a pointer to a native SQL store that can be used
+	// for native SQL queries for tables that already support it. This may
+	// be nil if the use-native-sql flag was not set.
+	NativeSQLStore *sqldb.BaseDB
```

Comment:
Same comment here re using either interface instead.

Reply:
See below, where we construct the invoice db. As we add more SQL based tables, this same dependency could be used for all of them (given they live in the same db file).